cmd/payload-testing-ui/server.go

```diff
@@ -31,38 +48,57 @@ const (
 	runsListTemplate = `
 <h1>Pull Request Payload Qualification Runs</h1>
 {{ len .Items }} run(s)
-<ul>
-{{ range .Items }}
-  <li>
-    {{ with .ObjectMeta }}
-      {{ with $url := printf "%s/%s" .Namespace .Name }}
-        <a href="` + runsURL + `{{ $url }}">{{ $url }}</a>
-      {{ end }}
-    {{ end }}
-  </li>
-{{ end }}
-</ul>
+<table class="table">
+	<thead>
+		<tr>
+			<th title="The name of the Pull Request Payload Qualification Run" class="info">Name</th>
+			<th title="The repository of pull request" class="info">Repository</th>
+			<th title="Hovering over the number will display the name of the pull request" class="info">Pull Request</th>
+		</tr>
+	</thead>
+	<tbody>
+		{{ range .Items }}
+		<tr>
+			<td>
+			<nobr>
```

Comment:
[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr) says this element type is deprecated and recommends the `white-space: nowrap` CSS attribute.

Looks like `bootstrap` already has this, so we can just add it to the `a` element: https://getbootstrap.com/docs/5.3/utilities/text/#text-wrapping-and-overflow.